community/case-studies/wayfair/data-distribution.md

```diff
@@ -0,0 +1,165 @@
+consolidate knowledge and tune to a unified operating experience and
+center of excellence in our production estate based on a single well
+known and cost efficient easy to operate alternative.
+
+### Characteristics
+
+It is understood that data will be captured from and distributed to a
+diverse and ever-changing set of production systems - built in many
+programming languages and environments and distributed to an unknowable
+set of destination systems.  
+  
+These systems include, but are not limited to: Graylog Extended Log
+Format, Syslog,  
+Kafka, TCP, UDP, Telegraf, Influx Line Protocol, ElasticSearch and many
+more...
+
+*Connectivity must be flexible enough to capture data from and
+distributed data to  
+synchronous, asynchronous protocols, transports and systems without
+developers  
+needing to learn or integrate with a new technology. The solution must
+be invisible to  
+developers.*
+
+The unit economics of the existing system is hard to measure, hard to
+plan and hard to remediate; especially given the rising year on year
+firehose volumetrics, with rising velocity and rising peak loads.
+
+*99% of the data either originates as JSON, or is transformed to JSON
+just in time.*
+
+The target systems for displacement and replacement have very well
+understood and extract, load, transform, enrichment and normalization
+algorithms that are coded and configured differently in each system.
+This is difficult to maintain, enhance and evolve in an environment
+where infrastructure and services are continuously evolving.
+
+A rich set of ETL and data manipulation and mutation primitives are
+required so that all  
+existing methods used for data processing across the target systems can
+be replaced by  
+a single easy to understand, easy to program and optimized alternative.
+
+### Solution
+
+Tremor has already established itself in the traffic shaping domain in
+the logging and metrics domains - adding just-in-time traffic shaping
+capabilities when needed; and preserving the original system’s
+operational semantics and behaviours during normal operating conditions
+when the system is operating within planned capacity constraints.
+
+<img src="./media/data-distribution/image2.png" />  
+The mandate now expands to replacing many of our event capture systems
+deployed on on-premise and cloud-native operating environments across
+multiple data-centres globally.
+
+Instead of our operations and service reliability teams needing to learn
+and manage the service deployment and administrative lifecycle of
+multiple components in this evolution we displace and replace those
+systems - typically operating as sidecars on source systems where our
+applications run on virtual machines, bare metal or containerized
+environments with tremor.
+
+The relatively primitive domain specific language developed as part of
+the initial solution for traffic shaping is replaced with a new domain
+specific language \`tremor-script\` designed for expressive data
```

Comment:
there are a number of places where gdocs seems to have escaped the ticks:
```
\`tremor-script\`
```

should probably be:
```
`tremor-script`
```

(same in a few more places, probably search&replaceable :D)

Reply:
Another `pandoc` artefact!